### ISSUE TITLE
Database Wiped

### ISSUE THREAD

**Issue by milad-soufastai** [Sonarr/Sonarr#2767]
**Describe the bug**
I've had this issue since June where my database would get corrupted like once every 2 weeks or so causing me to have to restore from a backup. I upgraded to the Phantom Branch and the problem had seemed to die down but tonight it was back again.

**Screenshots**

**Logs**
https://pastebin.com/gsGesWie

**System Information**

 - Sonarr Version: [e.g. 2.0.0.1] 3.0.0.266
 - Operating System: [e.g. Windows 10] Windows Server 2012 R2 Essentials
 - .net Framework (Windows) or mono (macOS/Linux) Version: [e.g. 4.5 or 5.12] .net 2.0 - .net 4.0

**UI Bugs:**
 - OS: [e.g. Windows]
 - Browser: [e.g. chrome, firefox]
 - Version: [e.g. 22]

**Additional context**

**Comment by markus101** [comment]
I don't see any indication of database corruption, database corruption won't result in all series being removed. It looks like someone has removed all your shows, either because your Sonarr install is wide open to the world or they guessed your password.

Reset you password (or set one if you don't have one) and reset the API key.

**Comment by milad-soufastai** [comment]
@markus101 I have my entire Sonarr instance sitting behind an SSL Enabled Reverse Proxy and my Proxy Access Logs don't have any indication of someone else accessing my instance outside of myself

**Comment by milad-soufastai** [comment]
and yes it is password protected as well

**Comment by milad-soufastai** [comment]
@markus101 For the sake of argument though, I reset my password, reset my API key, and turned on trace logging so that the next time this happens, it's better collected but some mysterious person who has a vendetta against my Sonarr instance who once a month deletes all my shows seems like it might not be the true reason why this is happening

**Comment by markus101** [comment]
```
18-10-10 20:31:40.7|Error|CommandExecutor|Error occurred while executing task SeasonSearch
 
[v3.0.0.266] NzbDrone.Core.Datastore.ModelNotFoundException: Series with ID 113 does not exist
```

Did you start a series search for multiple series or a missing search? There are a number of series that are being searched for and Sonarr wouldn't search for those on it's own. The only automated searches it does is when a series is added (and search for missing is checked) or when a download fails, but that'd be for a single episode.

Trace/Debug logs should show any API calls for deletions.

**Comment by milad-soufastai** [comment]
I did not start any series search. I logged into Sonarr and at the bottom left it was just showing Failed so I went into the logs and saw the series missing error message so I went into the UI and saw that none of my series were there. My sonarr database was also tiny compared to the backup. Interestingly, my debug logs show Sonarr searching for a lot of series that I don't have (Is this Sonarr just processing the RSS feed?):

18-10-10 21:34:09.1|Debug|ParsingService|No matching series Its Always Sunny in Philadelphia
18-10-10 21:34:09.1|Debug|DownloadDecisionMaker|Release rejected for the following reasons: [Permanent] Unknown Series
18-10-10 21:34:09.1|Debug|DownloadDecisionMaker|Processing release 'Its.Always.Sunny.in.Philadelphia.S13E06.720p.WEBRip.x264-TBS' from 'Omgwtfnzbs'
18-10-10 21:34:09.1|Debug|Parser|Parsing string 'Its.Always.Sunny.in.Philadelphia.S13E06.720p.WEBRip.x264-TBS'
18-10-10 21:34:09.1|Debug|Parser|Episode Parsed. Its Always Sunny in Philadelphia - S13E06 
18-10-10 21:34:09.1|Debug|Parser|Language parsed: English
18-10-10 21:34:09.1|Debug|QualityParser|Trying to parse quality for Its.Always.Sunny.in.Philadelphia.S13E06.720p.WEBRip.x264-TBS
18-10-10 21:34:09.1|Debug|Parser|Quality parsed: WEBRip-720p v1
18-10-10 21:34:09.1|Debug|Parser|Release Group parsed: TBS
18-10-10 21:34:09.1|Debug|ParsingService|No matching series Its Always Sunny in Philadelphia
